Core/Controllers/Api/EntryApiController.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.DependencyInjection;
+
+using VoteMyst.Database;
+using VoteMyst.Controllers.Api.Models;
+using Microsoft.AspNetCore.Http;
+
+namespace VoteMyst.Controllers.Api
+{
+    public class EntryApiController : VoteMystApiController
+    {
+        private ILogger _logger;
+
+        public EntryApiController(IServiceProvider serviceProvider) : base(serviceProvider)
+        {
+            _logger = serviceProvider.GetService<ILogger<EntryApiController>>();
+        }
+
+        [HttpPost]
+        [Route("api/entry/{id}/delete")]
+        public IActionResult Delete(string id)
+        {
+            throw new NotImplementedException();
```

Comment:
Requires implementation before public release; legal reasons.

Reply:
Obviously, it's currently WIP with #122.